src/gemini/epics/jms/JmsEpicsManager.cpp

```diff
@@ -27,7 +27,7 @@ JmsEpicsManager::JmsEpicsManager() throw (CommunicationException) {
 		throw CommunicationException("Trouble initializing request producer :"
 				+ e.getMessage());
 	} catch (TimeoutException &e) {
-		LOG4CXX_WARN(logger, "Can't get valid epics channel names, trying later...")
+		LOG4CXX_WARN(logger, "Can't get valid epics channel names, trying later...");
```

Comment:
Because we were using the apache-log4cxx-0.11.0 instead of apache-log4cxx-0.10.0. The macro implementation changed and now we need to specify the semicolon. If you want to see more details you can check the src/main/include/log4cxx/logger.h in  apache-log4cxx-0.11.0 and  apache-log4cxx-0.10.0.